include/fty_common_mlm_pool.h

```diff
@@ -182,13 +182,12 @@ class Pool
     unsigned size() const
     {
         std::lock_guard<std::mutex> lock(m_mutex);
-        return m_freePool.size();
+        return static_cast<unsigned>(m_freePool.size());
     }
 
     unsigned getCurrentSize() const
     {
-        std::lock_guard<std::mutex> lock(m_mutex);
-        return m_freePool.size();
+        return size();
```

Comment:
Why two functions which do the same thing ?

Reply:
a good question. I've no idea  ;)

Reply:
> Why two functions which do the same thing ?
a good question. I've no idea ;)